libcudacxx/test/libcudacxx/std/numerics/bit/bit.cast/bit_cast.pass.cpp

```diff
@@ -0,0 +1,348 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+// <cuda/std/bit>
+//
+// template<class To, class From>
+//   constexpr To bit_cast(const From& from) noexcept;
+
+#include <cuda/std/array>
+#include <cuda/std/bit>
+#include <cuda/std/cassert>
+#include <cuda/std/cmath>
+#include <cuda/std/cstdint>
+#include <cuda/std/limits>
+
+#include "test_macros.h"
+
+__host__ __device__ cuda::std::size_t memcmp(void* lhs, void* rhs, size_t bytes) noexcept
```

Comment:
Question: doesn't this symbol clash with `memcmp` from the C library? I would use a different name here.